tests/testStaticBrcode.js

```diff
@@ -0,0 +1,52 @@
+const assert = require('assert');
+const starkinfra = require('../index.js');
+const {generateExampleStaticBrcodeJson} = require('./utils/staticBrcode');
+
+starkinfra.user = require('./utils/user').exampleProject;
+
+describe('TestStaticBrcodePost', function(){
+    this.timeout(10000);
+    it('test_success', async () => {
+        let brcodes = [];
+        brcodes.push(new starkinfra.StaticBrcode(generateExampleStaticBrcodeJson()));
+        brcodes = await starkinfra.staticBrcode.create(brcodes);
+        for (let brcode of brcodes) {
+            assert(typeof brcode.uuid == 'string');
+        }
+    });
+});
+
+describe('TestStaticBrcodeQueryGet', function(){
+    this.timeout(10000);
+    it('test_success', async () => {
+        let i = 0;
+        const brcodes = await starkinfra.staticBrcode.query({limit: 5});
+        for await (let brcode of brcodes) {
+            assert(typeof brcode.uuid == 'string');
+            brcode = await starkinfra.staticBrcode.get(brcode.uuid);
+            assert(typeof brcode.uuid == 'string');
+            i += 1;
+        }
+        assert(i === 5);
```

Comment:
Acho feio isso de usar i. Não funciona pegar o tamanho do array no final?

Reply:
No node eu teria que fazer um for loop fazendo append em uma lista, para transformar o generator em uma lista, então no final daria no mesmo. mas posso fazer se preferir

Reply:
Mais bonito mesmo

Reply:
check